chapters/my_first_jina_app/index.md

```diff
@@ -0,0 +1,389 @@
+# Build your first Jina app
```

Comment:
The file should be in chapters/my_first_jina_app.md

Reply:
and images added to an image folder in the root

Reply:
Moved the markdown file. However I don't see any `images` folder in root of master branch. Should I create it myself?

Reply:
It's part of an ongoing PR #98 . I suppose you can wait to merge or add it yourself

Reply:
I'll hold off until it's merged

Reply:
I have reviewed it but you can hold off